src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -169,6 +178,47 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         render(<GeoChart {...geoPushpinProps} />, document.querySelector(geoPushpinElement));
     }
 
+    protected handleCenterPositionChanged = (data: IGeoChartProperties): void => {
+        const onCenterPositionChanged = get(this.callbacks, "onCenterPositionChanged", noop) as any;
+        const resultingData = data;
```

Comment:
We dont need to create new `resultingData ` var here.

Reply:
updated, change logic to AD